Cargo.toml

```diff
@@ -30,13 +29,23 @@ num-derive = "0.2"
 bitflags = "1.0.4"
 serial = { version = "0.4", optional = true }
 serde = { version = "1.0.101", optional = true }
-
-[dependencies.byteorder]
-version = "0.5.3"
-optional = true
-
+byteorder = { version = "0.5", optional = true }
 
 [features]
+"ardupilotmega" = []
+"asluav" = []
+"autoquad" = []
```

Comment:
It's missing `common`

Reply:
`common` should also be an default feature.

This also will fix `cargo test`, it's right now failing, it only works with all features enabled.

Reply:
There's no `common` feature. The common message set can't be disabled, since all other message sets depend on it anyway.

Reply:
The reason that the tests fail without features enabled is that I add tests that test whether messages are encoded and decoded properly in a specific (not `common`) message set, so the features that correspond to that message set need to be enabled.

Reply:
> There's no `common` feature. The common message set can't be disabled, since all other message sets depend on it anyway.

That's not true, `icarous` and `minimal` does not have `common` in `include` inside the `.xml`.

 

> The reason that the tests fail without features enabled is that I add tests that test whether messages are encoded and decoded properly in a specific (not `common`) message set, so the features that correspond to that message set need to be enabled.

This tests should have `#[cfg(feature = "message_set")]` to only enable with the feature selected.

Reply:
Ah, I didn't notice this was the case. I've added a `common` feature and feature-gated the tests that require the `ardupilotmega` feature.